### ISSUE TITLE
Background on colored tabs should be lighter for accessibility

### ISSUE THREAD

**Issue by angdesign** [microsoft/azuredatastudio#3293]
Issue Type: <b>Bug</b>

Tabs that pull color into it have a background that fails accessibility contrast (4.5:1)

![image](https://user-images.githubusercontent.com/29785331/48809266-c1019e80-ecd8-11e8-972d-1abc4d5c7474.png)

Recommendation is to change the strength of the tab background from 50%:
![image](https://user-images.githubusercontent.com/29785331/48809289-e0003080-ecd8-11e8-8110-45f4e434d594.png)

to 10%:
![image](https://user-images.githubusercontent.com/29785331/48809299-eb535c00-ecd8-11e8-94b1-4c1fd3bf3d24.png)

Which results in a cleaner look that passes accessibility:
![image](https://user-images.githubusercontent.com/29785331/48809406-513fe380-ecd9-11e8-8c40-0dc3225ff0ec.png)

![image](https://user-images.githubusercontent.com/29785331/48809429-687ed100-ecd9-11e8-830b-1dbc9869e7fb.png)

Azure Data Studio version: Azure Data Studio 1.2.4 (1ebfd0235690ad044ed2874a02e65188f5fea01c, 2018-11-02T21:36:55.143Z)
OS version: Darwin x64 18.2.0

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-4770HQ CPU @ 2.20GHz (8 x 2200)|
|GPU Status|2d_canvas: enabled<br>checker_imaging: disabled_off<br>flash_3d: enabled<br>flash_stage3d: enabled<br>flash_stage3d_baseline: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>native_gpu_memory_buffers: enabled<br>rasterization: enabled<br>video_decode: enabled<br>video_encode: enabled<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|2, 2, 2|
|Memory (System)|16.00GB (1.04GB free)|
|Process Argv|/Applications/Azure Data Studio.app/Contents/MacOS/Electron|
|Screen Reader|no|
|VM|0%|

</details><details><summary>Extensions (1)</summary>

Extension|Author (truncated)|Version
---|---|---
sql-vnext|Mic|0.8.0

</details>

**Comment by kburtram** [comment]
Closing older issues.  Please use the High Contrast theme if there are any color visibility issues.  Thanks!
